### DISCUSSION TITLE
About overview page.!!!

### DISCUSSION THREAD

**Discussion by Yasitha-Bhanuka** [community/community#60636] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

In order to create a **perfect overview page with animations** , the user must have good _HTML_ knowledge, right?

**Comment by GhostOf0days** [comment, marked as answer]
No. You can create code in other repos that create something then serve it dynamically in a website, fetching that content from the website for your ReadMe. You have to know CSS well, however. The other option is just pasting a GIF in your ReadMe.

**Reply by Yasitha-Bhanuka** [reply]
Yeah. You are correct. Link suitable GIF in to the ReadMe the easy way for make the better overview page. Thank you.
